src/yukon/backend/src/devserv/description/template/sess_description.json

```diff
@@ -0,0 +1,61 @@
+{
+        "log_file": "",
+        "interactions": [{
+                "bus": {
+                        "id": ""
+                },
+                "nodes": {
+                        "0": {
+                                "subject": {
+                                        "publish": {
+                                                "timestamp_start": "",
+                                                "timestamp_end": "",
+                                                "port_id": "",
+                                                "timeout": ""
+                                        },
+                                        "subscribe": {
+                                                "timestamp_start": "",
+                                                "timestamp_end": "",
+                                                "port_id": "",
+                                                "timeout": ""
+                                        }
+                                },
+                                "service": {
+                                        "send": {
```

Comment:
Services can be invoked (called) or provided (served). Sending/receiving pertains to very low-level interaction, we don't deal with that at the presentation layer (i.e., here).